lib/api_callable.js

```diff
@@ -190,19 +190,32 @@ function retryable(aFunc, retry, otherArgs) {
   return function retryingFunc(argument, callback) {
     var canceller;
     var timeoutId;
-    var error = new Error('Retry total timeout exceeded before any' +
-        'response was received');
+    var error = new Error();
```

Comment:
What should be done about line 231 below  which adds a note to the error variable?